app/models/sentence.rb

```diff
@@ -7,6 +7,12 @@ class Sentence < ActiveRecord::Base
   has_many :translated_sentences, dependent: :destroy
   default_scope where(imported: false, type: nil)
 
+  attr_accessible :locid
+
+  def self.find_with_locid(locid, _iri=nil)
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.